index.js

```diff
@@ -1,17 +1,61 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [],
+    this.length = 0
```

Comment:
in this case you could have assigned it to this.items.length directly.

its correct that you assign it to 0, but ideally the length is tied up with the items